RELEASES.md

```diff
@@ -4,20 +4,42 @@ Frank!Framework release notes
 [Tags](https://github.com/frankframework/frankframework/releases)
 [JavaDocs](https://javadoc.frankframework.org/)
 
-Upcoming (8.0)
+Upcoming (8.1)
 --------------
-[Commits](https://github.com/frankframework/frankframework/compare/v7.9-RC1...HEAD)
+[Commits](https://github.com/frankframework/frankframework/compare/v8.0.0...HEAD)
+
+Requires JDK 17
+
+### Non backwards compatible changes
+- Larva package is renamed from `testtool` to `larva`. References inside the Larva property files to the `testtool` package should be updated to larva. Such as: `org.frankframework.testtool.FileSender` -> `org.frankframework.larva.FileSender`. It still works with the old package name in 8.1, as a compatibility feature. 
+
+8.0
+--------------
+[Commits](https://github.com/frankframework/frankframework/compare/v7.9-RC1...v7.9.0)
+
+Requires JDK 11
```

Comment:
```suggestion
Requires JDK 11 or later
```

Reply:
While typing, I also considered the `or later` part, but we decided to not release that version with higher Java versions. Some things we fixed later on Java 17, so to avoid questions on version 8.0 and developers running Java 21 with that, I prefer that version only. 
Or should we add more details, like 11 - 17, but some Larvas failed on Java 17, I remember.

Reply:
We did have the Maven build working with JDK17 and 21, but were not entirely sure about IAF-Test. In the end, I think it turned out that for IAF-Test all that needed fixing was just one single test-case, which needed "order by" to be applied to a query.

So that's why I think that also for 8.0, we can say "11 or later".